contracts/EthereumLightClient.sol

```diff
@@ -87,12 +92,38 @@ contract EthereumLightClient is Ethash, Initializable, PausableUpgradeable {
         _setFirstBlock(storedBlock);
     }
 
+    function _deleteBlock(uint256 blockNo) private {
+        uint256[] storage blockHashes = blocksByHeight[blockNo];
+        for(uint256 j = 0; j < blockHashes.length; j++) {
+            uint256 blockHash = blockHashes[j];
+            delete blocks[blockHash];
+            delete blockExisting[blockHash];
+        }
+        delete blocksByHeight[blockNo];
+        delete blocksByHeightExisting[blockNo];
+        delete finalizedBlocks[blockNo];
```

Comment:
need to remove after https://github.com/harmony-one/horizon/pull/31/files

Reply:
noted